### ISSUE TITLE
Check before to see if command is available

### ISSUE THREAD

**Issue by bucaran** [daenney/fisherman-pyenv#1]
```
fish: Unknown command 'pyenv'
~/.config/fisherman/conf.d/pyenv.fish (line 1): pyenv root
                                                ^
in command substitution
	called on line -1 of file ~/.config/fisherman/conf.d/pyenv.fish
```

So, check before to see if `pyenv` is available?

```fish
if not command -s > /dev/null
    echo "pyenv : Command not available. See github.com/yyuu/pyenv"
    exit 1
end
```

Same for `rbenv` I guess.

**Comment by daenney** [comment]
Fixed in c8c3d4c0247ddd8b6ed954 and also in the `rbenv` version.

**Comment by bucaran** [comment]
Thanks.
